infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackService.kt

```diff
@@ -18,7 +22,6 @@ import org.springframework.transaction.annotation.Transactional
 class LocationTrackService(
     dao: LocationTrackDao,
     private val alignmentService: LayoutAlignmentService,
-    private val switchService: LayoutSwitchService,
```

Comment:
Funktioiden siirtelyä, jonka seurauksena käänsin myös tämän riippuvuuden switch serciestä tännepäin. Toiminnallisesti ei mitään uutta.